lib/usage.dart

```diff
@@ -79,12 +80,16 @@ void _activePaneItemChanged(_) {
   if (editor == null || editor.getPath() == null) return;
 
   String path = editor.getPath();
-  int index = path.lastIndexOf('.');
-  if (index == -1) {
-    _ga.sendScreenView('editor');
-  } else {
-    path = path.substring(index + 1);
-    _ga.sendScreenView('editor/${path.toLowerCase()}');
+
+  // Ensure that the file is Dart related.
+  if (isDartFile(path) || projectManager.getProjectFor(path) != null) {
+    int index = path.lastIndexOf('.');
+    if (index == -1) {
+      _ga.sendScreenView('editor');
+    } else {
+      path = path.substring(index + 1);
+      _ga.sendScreenView('editor/${path.toLowerCase()}');
```

Comment:
This is the file type - we strip everything but the extension. So `editor/dart`, `editor/css`, ...

Totally agree that the full path would be too invasive.